packages/sw-appcache-behavior/build.js

```diff
@@ -32,7 +32,7 @@ module.exports = () => {
           commonjs(),
         ],
       },
-      outputName: 'build/client-runtime.js',
+      buildPath: 'build/client-runtime.js',
```

Comment:
NIT: should these be swapped out with a value in package.json or does that not apply in this package?